### ISSUE TITLE
New Loss Function

### ISSUE THREAD

**Issue by Airliin** [MIC-DKFZ/MultiTalent#9]
Hello, may I ask which file is the new loss function mentioned in the paper?

**Comment by constantinulrich** [comment]
Hi, you can find the implementation [here](https://github.com/MIC-DKFZ/MultiTalent/blob/0bc7b2c32e31cb00008354f2420b65b85ef09578/nnunet/training/network_training/custom_trainers/MultiTalent/MultiTalent/MultiTalent_Trainer_DDP.py#L544).

Compared to the default nnU-Net loss implementation we calculate the BCE and Dice loss only for annotated classes in the input batch. Therfore, we pass the information of the dataset origin into the loss function. 

Best, Constantin
